source/style-guide/style/text-formatting.txt

```diff
@@ -6,26 +6,6 @@ Text Formatting
 
```

Comment:
There's some broken stuff on this page in staging.. we'll probably want to take a look more in depth here

Reply:
hm, when I run my link checker, I don't get any 404's. What is an example you're seeing?

Reply:
The tip boxes just say "See also: " with no link or anything, and a few of the code blocks (under the Area names and Argument names sections) are empty.